### PR TITLE
III-5108 Document online events

### PR DESCRIPTION
**Pull request by bertramakers**
### Added

- Added documentation for online events

---

Ticket: https://jira.uitdatabank.be/browse/III-5108

### CODE REVIEW

File: projects/uitdatabank/docs/entry-api/events/create-online.md [hunk 1]
```diff
@@ -1,13 +1,224 @@
...
+
+## Updating an event's attendanceMode
+
+You can always change an event's `attendanceMode` after its initial creation by [updating the event](./update.md).
+
+Keep the following restrictions in mind:
+
+* When changing the attendanceMode from `offline` or `mixed` **to** `online`, you must remove the `location` property from the event at the same time (or explicitly set it to the "nil location").
+* When changing the attendanceMode from `online` **to** `offline` or `mixed`, you must add a `location` property with an `@id` that links to the place where the event happens at.
+
```
Comment (corneelwille) [inline projects/uitdatabank/docs/entry-api/events/create-online.md:219]: It's not a restriction but maybe some behaviour / backend logic inside UDB that is worth mentioning:

When changing the attendanceMode from `online` or `mixed` to `offline` the `onlineUrl` will automatically be removed by UiTdatabank (if present)

**Review by corneelwille — APPROVED** [review]
Very clear, short yet complete guide about online events 👌

File: projects/uitdatabank/docs/entry-api/events/create-online.md (hunk 1 above)
Comment (bertramakers) [reply projects/uitdatabank/docs/entry-api/events/create-online.md:219]: Good point, I will add it 👍

Comment (bertramakers) [reply projects/uitdatabank/docs/entry-api/events/create-online.md:220]: See [e0d0912](https://github.com/cultuurnet/apidocs/pull/189/commits/e0d0912c03f6c541cf321bb8609f038e55247eea)

**Review by corneelwille — APPROVED** [review]

**Pull request merged**